mathics/builtin/comparison.py

```diff
@@ -349,17 +386,42 @@ def apply_other(self, x, y, evaluation):
 
 
 class Less(_InequalityOperator, SympyFunction):
+    """
+    <dl>
+    <dt>'Less[$x$, $y$]'
+    <dt>'$x$ < $y$'
+        <dd>yields 'True' if $x$ is known to be less than $y$.
+    <dt>'$lhs$ < $rhs$'
+        <dd>represents the inequality $lhs$ < $rhs$.
+    </dl>
+    """
     operator = '<'
     sympy_name = 'StrictLessThan'
 
 
 class LessEqual(_InequalityOperator, SympyFunction):
+    """
+    <dl>
+    <dt>'LessEqual[$x$, $y$]'
+    <dt>'$x$ <= $y$'
+        <dd>yields 'True' if $x$ is known to be less than or equal to $y$.
+    <dt>'$lhs$ <= $rhs$'
+        <dd>represents the inequality $lhs$ ≤ $rhs$.
```

Comment:
I think we have several options here - maybe we can experiment and see what works best.

For the PDF docs we can just pass the `$...$` construct directly to LaTeX (maybe after inserting `\text` or `\mathrm` commands to make multicharacter variables like `$expr$` look better).

MathJax can interpret TeX math commands, and we already pull it in to handle the examples, so for the HTML docs we could configure MathJax to recognise `$` as a delimiter and then just pass the `$...$` constructs through verbatim (maybe special-casing simple cases, so we don't use MathJax if we just need a superscript or subscript).

If we want to accept TeX syntax in the docs, the plaintext part might be difficult. A totally different approach might be to treat the `$...$` constructs as Mathics expressions, then just apply OutputForm/TeXForm/MathMLForm/etc to produce each type of output. This sounds like it might be tricky (we'd need to do evaluation in doc.py) but it might be the best long-term solution, because we'd be able to leverage more pre-existing code.